docs/ui/components/Footer/Footer.test.tsx

```diff
@@ -2,39 +2,38 @@ import { render } from '@testing-library/react';
 import Router, { NextRouter } from 'next/router';
 import * as React from 'react';
 
-import DocumentationFooter, { githubUrl } from './DocumentationFooter';
+import { Footer } from './Footer';
+import { githubUrl } from './utils';
 
 const mockRouter = (router: Partial<NextRouter>): NextRouter => ({ ...Router, ...router });
 
 describe('DocumentationFooter', () => {
   test('displays default links', () => {
     const router = mockRouter({ asPath: '/', pathname: '/example/' });
-    const { container } = render(<DocumentationFooter router={router} title="test-title" />);
+    const { container } = render(<Footer mockedRouter={router} title="test-title" />);
```

Comment:
I understand your concerns here, but tests are really flaky when running with ESM (not everything works as expected) and some code practices or decision will also differ due to ESM codebase. The reason why I have set on passing mocked router is that I just want to avoid adding a lot of complexity for a simple test case.

> In the website, we create a mockedRouter: https://github.com/expo/universe/blob/3eeeb9ee25b55143a69f6fb7ec7a7399c9bc6692/server/website/test-utils.tsx#L103-L126

I believe that this code has been inspired by this article:
* https://jamespotz.github.io/blog/how-to-mock-userouter

However, if you scroll beneath the snippet you can see the `jest.spyOn` approach, which in this case, in my opinion is better that recreating router mock as HoC and then handling it via custom render. The problem is that `jest.spyOn` currently doesn't work with ESM:
* https://github.com/facebook/jest/issues/12145

We can convert the test file to CJS and alter the code to fix that, it doesn't feel like it's a good approach, but can do that if you want to.